graphql-ballerina/modules/commons/common_utils.bal

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+
+isolated function getRootOperationTypeName(RootOperationType operationType) returns string {
+    match operationType {
+        MUTATION => {
+            return "Mutation";
+        }
+        SUBSCRIPTION => {
+            return "Subscription";
+        }
+    }
+    return "Query";
+}
```

Comment:
Can't do this ?

match operationType {
        MUTATION => {
            return "Mutation";
        }
        SUBSCRIPTION => {
            return "Subscription";
        }
       _ => {
            return "Query";
       }
    }

Reply:
This function used sometime before, but now we don't need it. :D Removing it altogether.